source/Stargate-Model/NeoJSONWriter.extension.st

```diff
@@ -0,0 +1,35 @@
+Extension { #name : #NeoJSONWriter }
+
+{ #category : #'*Stargate-Model' }
+NeoJSONWriter >> mapZincEvent [
+
+	self for: ZnHeaders customDo: [ :mapping | 
+		mapping writer: [ :jsonWriter :headers | 
+			jsonWriter writeMap: headers ] ].
+	self for: ZnRequest do: [ :mapper | 
+		mapper
+			mapAccessor: #method;
+			mapProperty: #uri getter: [ :request | request uri printString ];
+			mapProperty: #body getter: [ :request | request contents ];
```

Comment:
We need to be careful here, I don't know if it's a good idea to just toss the contents of the request (a user-controlled thing) directly to the mapper. Probably we need to do something different for text and binary bodies and put an upper limit to the bytes to log.

Reply:
I agree. We need to think about the use cases and our broader context. I think that we can rely on a external component to do a detailed log about every request (too expensive at a smalltalk image level). Maybe we need just the  URI and method.